src/App.js

```diff
@@ -1,26 +1,37 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import Header from './Header';
+import Footer from './Footer';
+import styled, { createGlobalStyle } from 'styled-components';
+import ClassList from './ClassList';
 
-function App() {
+const GlobalStyles = createGlobalStyle`
```

Comment:
The majority of developers and styled components write "GlobalStyle" instead of "GlobalStyles. 
For a better view, maybe you could put your GlobalStyle into an extra file.